apps/etterlatte-vedtaksvurdering/src/test/kotlin/vedtaksvurdering/VedtakBehandlingServiceTest.kt

```diff
@@ -1128,11 +1129,17 @@ internal class VedtakBehandlingServiceTest {
         val behandlingId = randomUUID()
 
         coEvery { behandlingKlientMock.tilSamordning(behandlingId, attestant, any()) } returns true
-        coEvery { behandlingKlientMock.harEtterbetaling(behandlingId, attestant) } returns false
         coEvery { samKlientMock.samordneVedtak(any(), false, attestant) } returns true
 
         runBlocking {
-            repository.opprettVedtak(opprettVedtak(behandlingId = behandlingId, status = VedtakStatus.ATTESTERT))
+            repository.opprettVedtak(
+                opprettVedtak(
+                    behandlingId = behandlingId,
+                    status = VedtakStatus.ATTESTERT,
+                    soeker = Folkeregisteridentifikator.of("08815997000"),
```

Comment:
Kluss med testdata generert av øvrige tester (som alle benytter samme fnr), pga innføringen av `Vedtakstidslinje` nå. Finner ferdigstilte vedtak, men uten `VedtakFattet`-data som medfører NPE.

Reply:
Enklere enn å tømme aktuelle tabeller for hver test.